### ISSUE TITLE
Error while parsing configuration.json

### ISSUE THREAD

**Issue by Bboy486** [wyolum/TouchSelfie#42]
(Edited Issue)

I can launch the script within the TouchSelfie folder with this command: sudo ./photobooth.sh

1. When it launches and I press (using the 7" touchscreen) one of the options (https://prnt.sc/ukpis5) I do not get the countdown although it is in the config file. https://prnt.sc/ukpjk0

2. Also, I had to flip the touchscreen 180 degrees so the script is now shooting upsidedown. I did not see where I can edit that?

3. How do I set this to run on boot?

Update: This error below was because I was SSHing into the pi. Running the command via vnc worked. 

pi@raspberrypi:-/TouchSelfie/scripts $ python user_interface.py

Traceback (most recent call last):

File "user_interface.py", line 1152, in <module> ui = User Interface(config,window_size=(SCREEN SCREEN_H),

log_level = logging. DEBUG)

File "user_interface.py", line 140, in __init__ self.root = TKO

File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1828, in init self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)

_tkinter.TclError: no display name and no $DISPLAY environment

**Comment by wyojustin** [comment]
Thanks @Bboy486 , I guess you resolved your error.  We've had a lot of trouble keeping up with the Google API.  Can you state what you did to resolve the problem for he next time this comes up?

Justin

**Comment by Bboy486** [comment]
I did a clean install.

I flipped the display to fit the enclosure I bought (but I still have the issue of the camera orientation being -90 - I have an open issue still for that).


Still don't know how to run on boot. I created a bash file that can be double clicked but the RaspTouch screen doesn't have the best response so it doesn't always work from the screen.

With this I open up another issue on how to add a quit option to the gui. This is because it is very difficult to exit from touch (easy with a keyboard or vnc).
